modules/tinymce/src/plugins/table/test/ts/browser/TableSectionApiTest.ts

```diff
@@ -141,6 +173,25 @@ UnitTest.asynctest('browser.tinymce.plugins.table.TableSectionApiTest', (success
       ApiChains.cAssertContent(expectedContent)
     ]));
 
+  const cSwitchMultipleColumnsType = (startContent: string, expectedContent: string, command: string, type: string) =>
```

Comment:
The `type` here could be better constrained to `'td' | `'th'`. That would then remove the need for the `searchingForType` variable below.

Reply:
No, you misunderstand the use for the `searchingForType`. It is the opposite of `type`, as `type` is what it will be changed to, `searchingForType` is what it currently is.
I'll type-restict `type` though.